pokemongo_bot/websocket_remote_control.py

```diff
@@ -0,0 +1,39 @@
+import threading
+from socketIO_client import SocketIO, BaseNamespace
+
+
+class WebsocketRemoteControl(object):
+
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.host, port_str = self.bot.config.websocket_server_url.split(':')
+        self.port = int(port_str)
+        self.sio = SocketIO(self.host, self.port)
+        self.sio.on('bot:process_request', self.on_remote_command)
+
+    def start(self):
+        self.thread.start()
+        return self
+
+    def process_messages(self):
+        while True:
+            self.sio.wait(5)
+
+    def on_remote_command(self, command):
+        name = command['name']
+        command_handler = getattr(self, name)
```

Comment:
There should probably be a check that this is defined. Otherwise it seems like a security risk.